cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/UsingTest.scala

```diff
@@ -28,12 +28,17 @@ class UsingTest extends DocumentingTest {
     doc("Planner hints and the USING keyword", "query-using")
     initQueries(
       """FOREACH(i IN range(1, 100) |
-        |  CREATE (:Scientist {born: 1800 + i})-[:RESEARCHED]->(:Science)<-[:INVENTED_BY {year: 530 + (i % 50)}]-(:Pioneer {born: 500 + (i % 50)})-[:LIVES_IN]->(:City)-[:PART_OF]->(:Country {formed: 400 + i})
+        |  CREATE (:Scientist {born: 1800 + i})-[:RESEARCHED]->
+        |         (:Science)<-[:INVENTED_BY {year: 530 + (i % 50)}]-
+        |         (:Pioneer {born: 500 + (i % 50)})-[:LIVES_IN]->
+        |         (:City)-[:PART_OF]->
+        |         (:Country {formed: 400 + i, name:'ACountry' + i})
```

Comment:
That is nicer to read. Would you also consider changing `matchString`, for which I always have to scroll right?